README.md

```diff
@@ -84,91 +116,70 @@ npm run clean
 
 ### Each component includes
 
-1. Main JavaScript File
+1. Main JSX File
 2. SASS File
 3. Index File
 
 ### Each component optionally may include
 
 1. Folder with images
-2. Folder with icons
-
-Also, each component may include another component that follows all above listed rules.
+2. Another component that follows all listed above rules.
 
```

Comment:
Can we include optional utils file? That would come in handy when dealing with animations, heavy computing or specific content processing functions.

```md
3. Component-specific utils library, e.g. `component.utils.js`
```

Reply:
@BolotskyDev I would prefer to put it in the separate directory inside that component's folder, rather than the creation of extra file postfix.

Reply:
I don't know, extending tree in depth seems to me a bit more affecting structure simplicity action than creating a postfix. Anyway, the point is to explicitly describe developer's behavior in such cases, I am fine with both options 👍

Reply:
This is a good idea @BolotskyDev but I'm not sure yet how to do it better right now since we haven't had such case so far. Let's keep it in mind for now and when we have such case, we will decide how it is better to do?